Feature/20097 check submission microservice

danwilliams opened the pull request.

jon-shipley commented on check-submission/src/routes/index.ts:
```diff
@@ -0,0 +1,95 @@
+  }
+}
+
+// Create the HeroRouter, and export its configured Express.Router
```
HeroRouter?

GeorgeChatzigiannis commented on check-submission/src/services/psychometrician-util.service.ts:
```diff
@@ -0,0 +1,17 @@
+'use strict'
```
is Ps util service needed for the check submission microservice?

GuyHarwood commented:
@danwilliams your PRs keep failing the codacy checks (see above).  Can you ensure your editor respects the standardJS policy and this should stop this happening again.

GeorgeChatzigiannis commented on check-submission/src/spec/service/psychometrician-util.service.spec.ts:
```diff
@@ -0,0 +1,48 @@
+'use strict'
```
this file as well

GuyHarwood commented:
@GeorgeChatzigiannis now that we are consolidating all of the endpoints into one app, i will add a criteria to remove these files and also any other spurious ones.  As Dan is fairly new to the project it will be a lot more time consuming for him to identify these.  I will most likely take care of it once these are merged into master.

GeorgeChatzigiannis commented:
@GuyHarwood 👍

danwilliams commented:
@GuyHarwood the issue was that in the change to typescript, the linter was skipping the js files. This is what caused Codacy to complain about that single file. I fixed that gulp file and added standard when linting, for the .js files (at the moment only gulpfile, but maybe others will be added in the future) so the npm lint script now does `standard && tslint --project .`
